taskcluster/ci/toolchain/android.yml

```diff
@@ -7,6 +7,19 @@ job-defaults:
     worker:
         max-run-time: 1800
 
+linux64-android-gradle-repack:
+    description: "Android SDK (Linux) repack toolchain build"
+    run:
+        script: repack-gradle.sh
+        resources: []
```

Comment:
I suspect there are some files that this should track, I guess maybe `gradlew` and the `gradle` directory?